### ISSUE TITLE
Text direction stripped from location (terminal)

### ISSUE THREAD

**Issue by Two9A** [chubin/wttr.in#932]
I requested the weather for Talagang, Pakistan in iTerm2/OSX:

![Screenshot 2023-10-20 at 12 41 30](https://github.com/chubin/wttr.in/assets/235537/d7b82a25-caf6-47ae-bdaf-f7a7b1034a43)

The location is given in Urdu, but:
- The text is displayed left-to-right, but the character forms are correct for RTL text;
- When piped to a hex editor, there doesn't seem to be an RTL mark between "Location:" and the location text (or a corresponding mark before the lat/long);
- When copy-pasting into this issue report, the browser seems to have some heuristic regarding text from that Unicode page, so the location comes out as: 
- تلہ گنگ, ضلع اٹک, پنجاب, پاکستان [32.9235108,72.4139769]

To summarize: there's no RTL mark before RTL text in the location, so it's being output backwards.

**Comment by chubin** [comment]
Thanks a lot for reporting. The problem is understood.
